cypress/support/pages/product.pageObject.js

```diff
@@ -0,0 +1,17 @@
+/// <reference types="Cypress" />
+
+import PageObject from '../PageObject';
+
+class ProductPageObject extends PageObject {
+  selectProduct(productName) {
+    cy.contains('.hrefch', productName)
+      .click();
+  }
+
+  clickAddToCart() {
+    cy.contains('a', 'Add to cart')
+      .click();
+  }
+}
+
+export default ProductPageObject;
\ No newline at end of file
```

Comment:
There should be a newline at the end of the file. It is a common convention to put this newline in so that certain utilities such as cat, awk, etc. do not break.

Reply:
done